libsolidity/parsing/DocStringParser.cpp

```diff
@@ -72,10 +72,7 @@ string::const_iterator skipWhitespace(
 	string::const_iterator _end
 )
 {
-	auto currPos = _pos;
-	while (currPos != _end && (*currPos == ' ' || *currPos == '\t'))
-		currPos += 1;
-	return currPos;
+	return ranges::find_if_not(ranges::make_subrange(_pos, _end), [](char const& it) { return (it == ' ' || it == '\t'); });
```

Comment:
These expressions are much less readable if the lambda is defined inline.
```suggestion
	auto isWhitespace = [](char const& c) { return (c == ' ' || c == '\t'); };
	return ranges::find_if_not(ranges::make_subrange(_pos, _end), isWhitespace);
```

Reply:
done